lib/srv/db/cloud/watchers/rds.go

```diff
@@ -56,39 +56,36 @@ func (c *rdsFetcherConfig) CheckAndSetDefaults() error {
 	return nil
 }
 
-// rdsFetcher retrieves RDS databases.
-type rdsFetcher struct {
+// rdsDBInstancesFetcher retrieves RDS DB instances.
+type rdsDBInstancesFetcher struct {
 	cfg rdsFetcherConfig
 	log logrus.FieldLogger
 }
 
-// newRDSFetcher returns a new RDS databases fetcher instance.
-func newRDSFetcher(config rdsFetcherConfig) (Fetcher, error) {
+// newRDSDBInstancesFetcher returns a new RDS DB instances fetcher instance.
+func newRDSDBInstancesFetcher(config rdsFetcherConfig) (Fetcher, error) {
 	if err := config.CheckAndSetDefaults(); err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return &rdsFetcher{
+	return &rdsDBInstancesFetcher{
 		cfg: config,
 		log: logrus.WithFields(logrus.Fields{
-			trace.Component: "rds-watcher",
+			trace.Component: "watcher:rds:instances",
```

Comment:
Nit: I usually try to avoid long component names because they're getting truncated in the logs to 9 or 10 characters IIRC. Maybe use something shorter like `watch:rds` here and `watch:aurora` below?